CONTRIBUTING.md

```diff
@@ -42,7 +42,25 @@ Before submitting a pull request, consider the following guidelines:
 
 # Getting started
 
-If you are using VSCode, you can install the [Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension. Once installed you will be prompted to reopen the folder in a container. All required dependencies will be installed in the container for you. If you're not prompted, you can run the `Remote-Containers: Open Folder in Container` command from the [VSCode Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette).
+There are several ways to set up your local environment so that you are ready to build, test and contribute to the Qwik project.
+
+## The recommended way
+
+This is the best approach because all required dependencies will be installed in the docker container for you and without affect your personal configuration in any way.
```

Comment:
```suggestion
This is the best approach because all required dependencies will be installed in the docker container for you and won't affect your personal configuration in any way.
```